src/tcp.toit

```diff
@@ -31,3 +33,84 @@ class TcpTransport implements Transport:
       with_timeout timeout:
         return Packet.deserialize reader_
     return null
+
+/**
+A transport for backing an MQTT client with TCP or TLS/TCP.
+
+Supports reconnecting to the same server.
+*/
+class ReconnectTcpTransport implements ReconnectTransport:
```

Comment:
Maybe the TLS and non-TLS versions should be split into two classes?

Reply:
done.